### PR TITLE
Added support for showAdMarquee and showControlBar from skin-config.json

### PR DESCRIPTION
**Pull request by akshaydch11**

### CODE REVIEW

File: sdk/react/panels/videoView.js [hunk 1]
```diff
@@ -304,9 +304,8 @@ var VideoView = React.createClass({
        <ActivityIndicator
           style={loadingStyle}
           size="large"
-      />
-        );
-      }
+       />
+      );
```
Comment (pilievOoyala) [inline sdk/react/panels/videoView.js:308]: do we not need the extra '}' anymore?

Comment (akshaydch11) [reply sdk/react/panels/videoView.js:308]: No.. not needed ..  I changed some code in last pull request and missed to remove this one. So checking it in now:)

Comment (lental) [reply sdk/react/panels/videoView.js:308]: good catch peter :)

but looking at the previous commit (https://github.com/ooyala/native-skin/commit/fef9a92f6a300796fba70acc2911811dcb121f2f) I can see we removed a layer of conditions, but didn't remove the last brace like akshay said

**Comment by brol1dev** [comment]
LGTM

**Pull request merged**